cpp/test/neighbors/ann_cagra.cuh

```diff
@@ -190,18 +271,18 @@ class AnnCagraTest : public ::testing::TestWithParam<AnnCagraInputs> {
     std::cout << "Done.\nRuning rng" << std::endl;
     raft::random::Rng r(1234ULL);
     if constexpr (std::is_same<DataT, float>{}) {
-      r.uniform(database.data(), ps.n_rows * ps.dim, DataT(0.1), DataT(2.0), stream_);
-      r.uniform(search_queries.data(), ps.n_queries * ps.dim, DataT(0.1), DataT(2.0), stream_);
+      r.normal(database.data(), ps.n_rows * ps.dim, DataT(0.1), DataT(2.0), stream_);
```

Comment:
It will be probably more consistent to just remove `stream_` and use `handle.get_stream()` everywhere. I have added a commit into my follow-up PR: https://github.com/rapidsai/raft/pull/1494/commits/40ba2068fd9cd123d5449665e9e13dea98fac164

Reply:
I have fixed the test code to use handle.get_stream() instead of stream_ in AnnCagraSortTest.